### PR TITLE
Analyses bug

### PR DESCRIPTION
**Pull request by sri2k1us**
- Fix race condition when drilling down HT Analysis
- Display dot menu in mobile screen even when nothing is selected to enable filtering.

### CODE REVIEW

File: src/components/analyses/listing/Listing.js [hunk 1]
```diff
@@ -648,11 +640,9 @@ function Listing(props) {
     const getSelectedAnalyses = (analyses) => {
         const items = analyses ? analyses : selected;
         if (items) {
-            return items
-                .map((id) =>
-                    data?.analyses.find((analysis) => analysis.id === id)
-                )
-                .filter((item) => item !== null && item !== undefined);
```
Comment (sri2k1us) [inline src/components/analyses/listing/Listing.js:655]: Now that I fixed the race condition this `null` filtering is not required anymore 🤷🏽‍♂️

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

**Review by psarando — APPROVED** [review]
LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for reviews.

**Pull request merged**